sql/updates/world/047_Updates_14.08.16.sql

```diff
@@ -0,0 +1,193 @@
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21230 AND `item` = 24368; -- 16
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21229 AND `item` = 24368; -- 16
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21228 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21227 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21226 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21225 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21224 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21221 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21220 AND `item` = 24368; -- 12
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 0 WHERE `entry` = 21218 AND `item` = 24368; -- 12
+
+-- Mortog Steamhead 2.1 back
+UPDATE `creature` SET `spawnmask` = 0 WHERE `guid` = 84715; -- 1
+
+-- Primal Nether / Nether Vortex
+UPDATE `item_template` SET `flags`= 1 WHERE `entry` IN (30183,23572);
+
+-- Design: Veiled Nightseye
+INSERT IGNORE INTO `creature_loot_template` VALUES
+-- 0.03
+(21337,31878,0.75,0,1,1,0,0,0),
+-- 0.02
+(18320,31878,0.50,0,1,1,0,0,0),
+-- 0.01
+(16468,31878,0.25,0,1,1,0,0,0),
+(18493,31878,0.25,0,1,1,0,0,0),
+(20301,31878,0.25,0,1,1,0,0,0),
+(18497,31878,0.25,0,1,1,0,0,0),
+(20299,31878,0.25,0,1,1,0,0,0),
+(23397,31878,0.25,0,1,1,0,0,0),
+(23018,31878,0.25,0,1,1,0,0,0),
+(16481,31878,0.25,0,1,1,0,0,0),
+(16415,31878,0.25,0,1,1,0,0,0),
+(24685,31878,0.25,0,1,1,0,0,0),
+(24683,31878,0.25,0,1,1,0,0,0),
+(20259,31878,0.25,0,1,1,0,0,0),
+(17833,31878,0.25,0,1,1,0,0,0),
+(20530,31878,0.25,0,1,1,0,0,0),
+(21298,31878,0.25,0,1,1,0,0,0),
+(18323,31878,0.25,0,1,1,0,0,0),
+(20692,31878,0.25,0,1,1,0,0,0),
+(20035,31878,0.25,0,1,1,0,0,0),
+(20031,31878,0.25,0,1,1,0,0,0),
+(19166,31878,0.25,0,1,1,0,0,0),
+(21543,31878,0.25,0,1,1,0,0,0),
+(21695,31878,0.25,0,1,1,0,0,0),
+(21917,31878,0.25,0,1,1,0,0,0),
+-- 0.005
+(15547,31878,0.10,0,1,1,0,0,0),
+(15548,31878,0.10,0,1,1,0,0,0),
+(15551,31878,0.10,0,1,1,0,0,0),
+(16406,31878,0.10,0,1,1,0,0,0),
+(17148,31878,0.10,0,1,1,0,0,0),
+(18875,31878,0.10,0,1,1,0,0,0),
+(21911,31878,0.10,0,1,1,0,0,0),
+(22877,31878,0.10,0,1,1,0,0,0),
+(17895,31878,0.10,0,1,1,0,0,0),
+(16539,31878,0.10,0,1,1,0,0,0),
+(16409,31878,0.10,0,1,1,0,0,0);
+
+-- Landmark under A'dal
+UPDATE `gameobject` SET `spawnmask`=0 WHERE `guid` = 47204; -- 1
+
+-- Reduce Damage due to Core Timer Changes
+--
+-- High King Maulgar – Hochkönig Maulgar 18831
+UPDATE `creature_template` SET `mindmg`='10783',`maxdmg`='12805',`baseattacktime`='1500',`speed`='3.00' WHERE `entry` IN ('18831'); -- 10783 12805 1449 -- 20487 24330 -- 21565 25610 
```

Comment:
yea started wrting like this as it makes copy pastin more easy. same with '' and numeric insertions. will try to avoid that in the future.